src/server/protocol.ts

```diff
@@ -2493,18 +2493,6 @@ namespace ts.server.protocol {
         projectName: string;
     }
 
-    export type SurveyReadyEventName = "surveyReady";

```

Comment:
I am not sure if we need to keep this? For compatibility?

Reply:
That implies everything in this file, right? 

It does indeed look like VS Code still depends on this. (I mentioned removing the survey to Matt last month but didn't get a firm commitment on it.)
https://github.com/Microsoft/vscode/blob/248aea7fd51a703a7ab94ae5a582c85c11fbff33/extensions/typescript-language-features/src/typescriptServiceClient.ts